src/app/query/bitcoin/ordinals/use-inscriptions.query.ts

```diff
@@ -25,6 +25,14 @@ interface InscriptionsQueryResponse {
   total: number;
 }
 
+interface CurrentAccData {
+  [key: number]: {
+    addressesWithoutOrdinalsNum: number;
+    fromIndex: number;
+    addressesMap: Record<string, number>;
+  };
+}
```

Comment:
Would structure as just in case we ever need to use the inner type alone

```suggestion
interface AccountData {
  addressesWithoutOrdinalsNum: number;
  fromIndex: number;
  addressesMap: Record<string, number>;
};

type AccountDataMap =  Record<number, AccountData>;
```